src/olympia/versions/models.py

```diff
@@ -291,21 +291,22 @@ def from_upload(cls, upload, addon, selected_apps, channel,
 
         version_uploaded.send(instance=version, sender=Version)
 
-        try:
-            yara_result = YaraResult.objects.get(upload_id=upload.id)
-            yara_result.version = version
-            yara_result.save()
-        except YaraResult.DoesNotExist:
-            log.exception('Could not find a YaraResult for FileUpload %s',
-                          upload.id)
-
-        try:
-            ScannersResult.objects.filter(upload_id=upload.id).update(
-                version=version
-            )
-        except ScannersResult.DoesNotExist:
-            log.exception('Could not find ScannersResults for FileUpload %s',
-                          upload.id)
+        if version.is_webextension:
+            if waffle.switch_is_active('enable-yara'):
+                try:
+                    yara_result = YaraResult.objects.get(upload_id=upload.id)
+                    yara_result.version = version
+                    yara_result.save()
+                except YaraResult.DoesNotExist:
+                    log.exception('Could not find a YaraResult for FileUpload '
+                                  '%s', upload.id)
+
+            if (
+                    waffle.switch_is_active('enable-customs') or
+                    waffle.switch_is_active('enable-wat')
```

Comment:
unnecessary/unusual indentation?

Reply:
How should this be formatted? (it'll be so much better when we'll have Black :p)

Reply:
meh, I don't care actually.  Nothing to see here.